chandra_aca/aca_image.py

```diff
@@ -159,7 +161,8 @@ def __array_finalize__(self, obj):
     __add__ = _operator_factory('add')
     __sub__ = _operator_factory('sub')
     __mul__ = _operator_factory('mul')
-    __div__ = _operator_factory('div')
+    if not six.PY3:
+        __div__ = _operator_factory('div')
```

Comment:
This makes sense to me (as opposed to assigning it differently or sum such which would just be more confusing in the end).